docs/can-guides/contribute/code.md

```diff
@@ -68,151 +68,152 @@ Download the Firefox browser
+|   ├── test.html           — Main test page
 ```
 
-Generally, speaking, the most important files are:
+Generally speaking, the most important files are:
 
- - the main module -  `can-example.js`
- - the main test module - `test/can-example-test.js`
- - the test page - `test/test.html`
+ - the main module —  `can-example.js`
+ - the main test module — `test/can-example-test.js`
+ - the test page — `test/test.html`
 
-When fixing a bug or making a feature, we'll add a test in the main test module
-and update code in the main module and verify the tests are passing by running
+To fix a bug or making a feature, add a test in the main test module, update code in the main module, and then verify the tests are passing by running
 the test page.
 
 Some modules have multiple modules, test modules, and test pages.  These modules are
-commonly organized as __modlets__ where each folder will have its own main module, test module
+commonly organized as __modlets__ where each folder will have its own main module, test module,
 and test page:
 
 ```
-├── a-module/            - Module's modlet folder
-|   ├── a-module.js      - The module
-|   ├── a-module-test.js - The module's tests
-|   ├── test.html        - A test page that runs just the module's tests
+├── a-module/            — Module’s modlet folder
+|   ├── a-module.js      — The module
+|   ├── a-module-test.js — The module’s tests
+|   ├── test.html        — A test page that runs just the module’s tests
 ```
 
-Where possible, CanJS code is:
+Where possible, CanJS code uses:
 
 - Tabs not spaces
-- JSHinted
+- JSHint
 - CommonJS not ES6
-- Follows jQuery [coding conventions](https://contribute.jquery.org/style-guide/js/)
+- jQuery’s [coding conventions](https://contribute.jquery.org/style-guide/js/)
 
 
 ##  Make your changes
 
-Once you've figured out where you need to make changes, you'll want to complete the following steps
+Once you’ve figured out where you need to make changes, you’ll want to complete the following steps
 to make those changes and create a pull request so we can include your code in future releases:
 
 
 1. Create a new feature branch. For example, `git checkout -b html5-fix`.
 2. Make some changes to the code and tests.
-4. Run tests `npm test` and make sure they pass in all browsers.
-5. Update documentation if necessary.
+4. Run `npm test` to make sure the tests pass in all browsers.
+5. Update the documentation if necessary.
 6. Push your changes to your remote branch.  For example, `git push origin html5-fix`.
-7. Submit a pull request! Navigate to Pull Requests and click the 'New Pull Request' button. Fill in some
-   details about your potential patch including a meaningful title. When finished, press "Send pull request". The core team will be notified about your submission and let you know of any problems or targeted release date.
+7. Submit a pull request! On GitHub, navigate to Pull Requests and click the “New Pull Request” button. Fill in some
```

Comment:
I'm wary of using special quotes other than `'` or `"`.  `'` and `"` are easier to type and, at least for me, provide equivalently readable punctuation.  I don't want to be inconsistent.  I've noticed these changes mistakenly in code blocks, which breaks the code.